### PR TITLE
[FAAS] Auto submit disable on config page.

### PR DESCRIPTION
**Pull request by seanchoi-dev**
Resolves: [MWPW-128249](https://jira.corp.adobe.com/browse/MWPW-128249)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/tools/faas?martech=off
- after: https://faas-fix--milo--adobecom.hlx.page/tools/faas?martech=off

### CODE REVIEW

**Review by sukamat — APPROVED** [review]

**Review by honstar — APPROVED** [review]

File: libs/blocks/faas-config/faas-config.js [hunk 1]
```diff
@@ -425,8 +425,8 @@ const Configurator = ({ rootEl }) => {
 
   useEffect(() => {
     if (isFaasLoaded) {
-      initFaas(state, document.getElementsByClassName('faas')[0]);
       saveStateToLocalStorage(state);
+      initFaas(JSON.parse(localStorage.getItem(LS_KEY)), document.getElementsByClassName('faas')[0]);
```
Comment (chrischrischris) [inline libs/blocks/faas-config/faas-config.js:429]: Why the change to reading from localstorage?  Isn't `state` the exact same?

Comment (seanchoi-dev) [reply libs/blocks/faas-config/faas-config.js:429]: @chrischrischris This PR is trying to set state.as as false if it is configuration page.
By this change, the faas config renders form as `state.as` false even though user sets as true.
